openshift311/010-check-cluster.md

```diff
@@ -0,0 +1,54 @@
+## Create a StorageOS Cluster
+
+To create a StorageOS cluster a StorageOSCluster resource is created.
+
+A StorageOSCluster is a custom resource acted upon by the StorageOS operator.
+When the StorageOS operator detects a StorageOSCluster resource it will create
+a StorageOS cluster based on the StorageOSCluster resource.
+
+``
+oc create -f - <<END
+apiVersion: "storageos.com/v1"
+kind: StorageOSCluster
+metadata:
+  name: "example-storageos"
+spec:
+  secretRefName: "storageos-api"
+  secretRefNamespace: "default"
+  images:
+    nodeContainer: "storageos/node:1.2.0"
+  disableTelemetry: true
+END
+``{{execute}}
+
+Verify that the pod enters a running state
+`oc get pods -n storageos -w`{{execute}}
+
+## Check StorageOS status
```

Comment:
I would mention that StorageOS Pod will take at least a min to report healthy. StorageOS bootstraps, and discovers other nodes, etc.